c2corg_ui/static/js/feed.js

```diff
@@ -120,6 +160,52 @@ app.FeedController = function(appAuthentication, appApi, appLang, imageUrl, ngeo
   this.ngeoLocation = ngeoLocation;
 
   this.getDocumentsFromFeed();
+  this.getDocumentsFromForum();
+  this.feedColumnController();
+  /**
+   * @type {number}
+   * @public
+   */
+
+};
+
+
+/**
+* refresh the feed column according the width
+* @export
+*/
+app.FeedController.prototype.feedColumnController = function() {
```

Comment:
Ok for the rename, but this function can be have a lot of change in the future, (maybe complety remove with a css solution), i guess it's more readable to separe this code from the rest because the goal it's not to get the doc, only manage the look.